int64 item_id  # corresponds to some unique item identifier, likely produced by vision
int64 box_id  # 0 for small box, 1 for big box
geometry_msgs/Pose end_pose  # in box coordinates
